recipe/meta.yaml

```diff
@@ -35,6 +35,7 @@ test:
     - metomi.rose
   commands:
     - PAGER=cat rose stem --help
+    - pip check
```

Comment:
pip isn't bundled with python any more so can't do this without installing test deps

```suggestion
```